scripts/pkg/build_templates/opensearch/rpm/opensearch.rpm.spec

```diff
@@ -175,16 +169,27 @@ exit 0
 %attr(0644, root, root) %config(noreplace) %{_prefix}/lib/sysctl.d/%{name}.conf
 %attr(0644, root, root) %config(noreplace) %{_prefix}/lib/tmpfiles.d/%{name}.conf
 
+%dir %attr(750, %{name}, %{name}) %{data_dir}
+%dir %attr(750, %{name}, %{name}) %{log_dir}
+%dir %attr(750, %{name}, %{name}) %{pid_dir}
+
+# Permissions
+%defattr(-, root, root)
+
+# Root dirs/docs/licenses
+%dir %{product_dir}
```

Comment:
So we dont want opensearch:opensearch to own product_dir?

Reply:
Correct.  Just like the rest of the system, the user running the service should not need to (be able to) overwrite its files.  The default ownership `root:root` make sense.